OneWelcomeExampleApp/SupportingFiles/Info.plist

```diff
@@ -6,6 +6,30 @@
 	<string>$(DEVELOPMENT_LANGUAGE)</string>
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIcons</key>
```

Comment:
how does it work? it changes the icon dynamically even if the app is killed?

Reply:
No,  it changes when you ask for it. I did it in `WelcomeViewController` in `viewDidLoad`

Reply:
so the whole PR make no sense :)
the icon changes only when you open the app. If the app is closed and you (or the system) changes to dark theme the icon is still white, isn't it?

Reply:
go ahead

Reply:
Yup, but IFAIK there is no other way to change an icon. It still makes sense - the icon is updated asap == when is launched. Let me recheck our options.